dictionaries/ui/application/links/fr.dictionary.itop.links.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Copyright (C) 2013-2021 Combodo SARL
+ *
+ * This file is part of iTop.
+ *
+ * iTop is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * iTop is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ */
+
+// Display DataTable
+Dict::Add('FR FR', 'French', 'Français', array(
+	'UI:Links:ActionRow:detach'              => 'Détacher',
+	'UI:Links:ActionRow:detach+'             => 'Détacher cet objet',
+	'UI:Links:ActionRow:detach:confirmation' => 'Voulez-vous détacher <b>{item}</b> de l\'objet courant ?',
+	'UI:Links:ActionRow:delete'              => 'Supprimer',
+	'UI:Links:ActionRow:delete+'             => 'Supprimer cet objet',
+	'UI:Links:ActionRow:delete:confirmation' => 'Voulez-vous supprimer <b>{item}</b> de l\'objet courant ?',
```

Comment:
```suggestion
	'UI:Links:ActionRow:Detach'              => 'Détacher',
	'UI:Links:ActionRow:Detach+'             => 'Détacher cet objet',
	'UI:Links:ActionRow:Detach:Confirmation' => 'Voulez-vous détacher <b>{item}</b> de l\'objet courant ?',
	'UI:Links:ActionRow:Delete'              => 'Supprimer',
	'UI:Links:ActionRow:Delete+'             => 'Supprimer cet objet',
	'UI:Links:ActionRow:Delete:Confirmation' => 'Voulez-vous supprimer <b>{item}</b> de l\'objet courant ?',
```

Same as previous comment. Mind to update the rest of their usages.